pkgs/applications/networking/ouroboros/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
+, cmake
+, dnsutils
+, fetchgit
+, fuse
+, libgcrypt
+, openssl
+, pkg-config
+, protobuf
+, protobufc
+, systemd
+}:
+stdenv.mkDerivation rec {
```

Comment:
suggestion: Use [`finalAttrs`](https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes) to avoid `rec`